src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -355,6 +355,8 @@ protected static ISQLServerDataSource updateDataSource(String connectionString,
                             break;
                         case Constants.SEND_TEMPORAL_DATATYPES_AS_STRING_FOR_BULK_COPY:
                             ds.setSendTemporalDataTypesAsStringForBulkCopy(Boolean.parseBoolean(value));
```

Comment:
This needs a break at the end of the case statement, or it'll overflow into the next one.